### DISCUSSION TITLE
Example for using parquet files for datasource?

### DISCUSSION THREAD

**Discussion by cpolzer** [evidence-dev/evidence#2874] [Q&A] [answered]
Hello,

I am currently trying to use polars to create a parquet file from a massive json "datadump". For visualization I hoped i could use evidence. 

Beeing new to this, how would one configure a data source with a parquet file? The [CSV data source docs](https://docs.evidence.dev/core-concepts/data-sources/#csv-files) mention it is configured analogue to csv, but for me this is not working. 

Here is what i figured out might be needed to configure the datasource:
``` bash
➜  evidence-test git:(feature/test-evidence) ✗ tree sources/     
sources/
└── test_db
    ├── connection.yaml
    └── test.parquet

2 directories, 2 files
➜  evidence-test git:(feature/test-evidence) ✗ cat sources/test_db/connection.yaml 
# This file was automatically generated
name: test_db
type: csv
options:
  filename: test.parquet
```

I data source settings in the webapp verify the datasource as "Connection Successful!", but what ever query i try ,its not working - and in the schema viewer, nothing is shown?

Any help is appreciated, and lots of regards,

Chris

**Comment by archiewood** [comment, marked as answer]
There is an out of tree connector for parquet you can use:

https://github.com/archiewood/evidence-connector-parquet

We plan to first-party this connector soon but in the meantime you can install it as a plugin!
